dashboard/lib/policies/inline_answer.rb

```diff
@@ -0,0 +1,17 @@
+class Policies::InlineAnswer
+  def self.visible?(user, script_level)
+    return true if Rails.application.config.levelbuilder_mode
+    return false unless user
+    return false unless script_level
+
+    script = script_level.try(:script)
+    return true if user.authorized_teacher? &&
+      script &&
+      !script.professional_learning_course?
+
+    user_level = UserLevel.find_by(user: user, level: script_level.level)
+    return true if user_level.try(:readonly_answers)
```

Comment:
Maybe not something you need to be responsible for, but I'm really missing the "why" of this logic.  Inline answers are visible if the user is authorized _or_ if they have `readonly_answers`? Maybe because the latter does not explain itself very well.  Looking at the previous PR explains somewhat:

> Teachers can also put stages into a readonly mode in which students are able to view the answers.

I wish that was obvious from reading this code, though.

Reply:
Oh, that's helpful! I'll drop that in as a comment